hub/powertoys/quick-accent.md

```diff
@@ -68,9 +66,9 @@ From the Settings menu, the following options can be configured:
 | :--- | :--- |
 | Activation key | Either `Left/Right Arrow`, `Space` or `Left, Right or Space`. |
 | Character set | Only characters that are in the selected set will be shown. |
-| Toolbar location | Where on the screen you want the tool. |
+| Toolbar location | Position of the toolbar. |
 | Show the Unicode code and name of the currently selected character | Shows the Unicode code (in hexadecimal) and name of the currently selected character under the selector. |
-| Sort characters by usage frequency | Sorts the characters by usage frequency. |
+| Sort characters by usage frequency | |
```

Comment:
Do you want to leave the Description blank for this table entry?

Reply:
Yeah, when the description is the same text as the setting, it's not needed